client-sq/src/components/History.js

```diff
@@ -55,8 +66,141 @@ const History = ({ theme }) => {
   }
 
   return (
-
-
+    <div>
+    { isMobile ? (
```

Comment:
This might be rendering the wrong components. The text enlarges when the device is in mobile view